tests/unit/BaseApiAbstractTest.php

```diff
@@ -25,12 +25,7 @@ public function testNoUserAgentExtensions()
         $instance = FileApi::create($this->authProvider, 'test');
         $http_client = $this->invokeMethod($instance, 'getHttpClient');
 
-        $this->assertNotFalse(
-            \strpos(
-                $http_client->getConfig()['headers']['User-Agent'],
-                'smartling-api-sdk-php/3.9.2 (no extensions) GuzzleHttp/6'
-            )
-        );
+        $this->assertEquals('smartling-api-sdk-php/4.0.0 (no extensions) GuzzleHttp/7', $http_client->getConfig()['headers']['User-Agent']);
```

Comment:
Previously user agent also constained some php related info. That's why there was `strpos` call. Now it's just `'smartling-api-sdk-php/4.0.0 (no extensions) GuzzleHttp/7`?

Reply:
When I first ran the test, it failed with
```
Failed asserting that false is not false.
```
I've refactored so that in the future it would fail with
```
Failed asserting that two strings are equal.
Expected :'smartling-api-sdk-php/4.0.0 (no extensions) GuzzleHttp/7'
Actual   :'smartling-api-sdk-php/4.0.1 (no extensions) GuzzleHttp/7'
```
As for the php related info, it has most likely changed, because Guzzle 7 uses the following code to add its UA
```
    public static function defaultUserAgent(): string
    {
        return sprintf('GuzzleHttp/%d', ClientInterface::MAJOR_VERSION);
    }
```